### PR TITLE
Import OpenAddresses

### PR DESCRIPTION
**Pull request by antoine-de**
connector to import data from https://openaddresses.io/, so now you can have addresses from all around the world :world_map: 

you can use this to import french data too if you want, but beware that it's the [ban](https://adresse.data.gouv.fr/), not [bano](http://openstreetmap.fr/bano).

(and it's a bit cumbersome since openaddresse split the files by regions)

### CODE REVIEW

File: src/bin/openaddresses2mimir.rs [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        let addr_label = format!("{} ({})", addr_name, self.city);
+        let street_id = format!("street:{}", self.id); // TODO check if thats ok
+        let admins = admins_geofinder.get(&geo::Coordinate {
+            x: self.lat,
+            y: self.lon,
+        });
+
+        let weight = admins
+            .iter()
+            .find(|a| a.level == 8)
```
Comment (antoine-de) [inline src/bin/openaddresses2mimir.rs]: I don't like this (it has been copied from the bano import)

but I see 2 ways to better handle thoses:
* add a `city_level` in the cli args
* better handle the `city_level` in `osm2mimir`.
  we could tag the admin at import time, so only  `osm2mimir` has this logic and `bano2mimir` and `openaddresses2mimir` only check `a.level == AdminLevel::City`.
  this raise some questions, if you are interested I can create an issue

Comment (kinnou02) [reply src/bin/openaddresses2mimir.rs]: I think this should be handled in osm2mimir.
Later we could use another data source for admin/city

Comment (antoine-de) [reply src/bin/openaddresses2mimir.rs]: I totally agree with this (and I got ideas for the other admin data source)

I'll create an issue for this then

Comment (TeXitoi) [reply src/bin/openaddresses2mimir.rs]: That's OK for bano as city will always be 8.

I think short term can be an argument with default to 8. And open an issue to think about this problem.

File: src/bin/openaddresses2mimir.rs [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    let admins = rubber
+        .get_admins_from_dataset(dataset)
+        .unwrap_or_else(|err| {
+            info!(
+                "Administratives regions not found in es db for dataset {}. (error: {})",
+                dataset, err
+            );
+            vec![]
+        });
+    let admins_geofinder = admins.iter().cloned().collect();
```
Comment (TeXitoi) [inline src/bin/openaddresses2mimir.rs]:
```rust
let admins_geofinder = admins.into_iter().collect();
```

**Comment by TeXitoi** [comment]
You need to rebase, the merged version doesn't compile.

**Pull request merged**